feat: Option to change tag click behaviour

Computerdores opened the pull request:
### Summary

Resolves #944.

### Tasks Completed

<!-- No requirements, just context for reviewers. -->

-   Platforms Tested:
    -   [x] Windows x86
    -   [ ] Windows ARM
    -   [ ] macOS x86
    -   [ ] macOS ARM
    -   [ ] Linux x86
    -   [ ] Linux ARM
-   Tested For:
    -   [x] Basic functionality
    -   [ ] PyInstaller executable

CyanVoxel commented:
Looks good! Just needs some base strings in `en.json` and I think this should be good to go. Other than that, it *would* be ideal if the "Add to Search" functionality didn't add tags to searches they already exist in (thus killing the search), but I understand that since it's using a string manipulation approach that this would be a bit messy to tack that onto.

Computerdores commented:
Ok TIL that `A AND A` doesn't return anything that is definitely a bug.
As for the strings, I totally forgot to add those will do that really quick.

Computerdores commented:
I have added the missing strings and also found and fixed a bug because of which the settings dropdowns were always english even if the user set TS to be in some other language
